python/py-gpilab-framework/Portfile

```diff
@@ -40,7 +40,9 @@ patchfiles                  mainWindow.py.diff \
 if {${name} ne ${subport}} {
     depends_lib-append      port:py${python.version}-numpy \
                             port:py${python.version}-psutil \
-                            port:py${python.version}-qtpy
+                            port:py${python.version}-qtpy \
+                            port:py${python.version}-pyqt5 \
```

Comment:
Please keep the dependencies sorted alphabetically.

Reply:
Also you need to remove the trailing slash on the last one.

Reply:
*Should*; not need. ;) (Continues to the blank line below, but agreed, it shouldn't be there.)